Proposed changes to membership policies (#19)
zrgn/zrgn.github.io: index.md

## Membership
There is no single leader of the Zargon community. Zargon is a community run effort. All members are encouraged to participate in this effort.

The following guidelines are intended to allow us to maintain an inclusive environment where all members feel comfortable gaming with any other member of the group rather than fracturing in cliques. They are enumerated in detail so as to mitigate subjectivity in role assignment.

### Obligations
* Members are expected to maintain basic personal hygiene.
* Members are expected to be gracious and respect additional restrictions of
  venue
    - e.g. no smoking inside; no young children; allergy considerations.

### Active Members
* An active member is one who regularly does one or more of the following
  - Attends scheduled gaming or social events
  - Paricipates in virtual conversations
* Active members are noted by the **Zargonaut** role in the group discord.

### Contributing Members
* A contributing member is one who regularly does one or more of the following
  - Schedules game sessions
  - Schedules social events
  - Brings new games for the group to try
  - Teaches games at gaming evens

>   - Creates content to aid the group (e.g. rules handouts, character illustions, games storage)

Reviewer: character *illustrations*

* Contributing members are noted by the **Adept** role in the group discord.

### New Members
* Any active member in good standing may invite non-member to join social
  or gaming events, but take responsibility for their behavior.

>   - In the case of a virtual event, they will be assigned the **Attendee** role in the group discord.

Reviewer: - **Attendee** roles are temporary and should be removed at the end of the specific event or on a regular basis

^ otherwise attendees basically have normal access

Author: I'd think they would only get access to the voice channels and a single text channel. But I'll add that

Good humored, cooperative, competitive, consensual torture, for the purposes of
exploring strategies, design, progress and history of gaming

## Membership
There is no single leader of the Zargon community. Zargon is a community run
effort. All members are encouraged to participate in this effort.

The following guidelines are intended to allow us to maintain an inclusive
environment where all members feel comfortable gaming with any other member of
the group rather than fracturing in cliques. They are enumerated in detail so
as to mitigate subjectivity in role assignment.

### Obligations
* Members are expected to maintain basic personal hygiene.
* Members are expected to be gracious and respect additional restrictions of
  venue
    - e.g. no smoking inside; no young children; allergy considerations.

### Active Members
* An active member is one who regularly does one or more of the following
  - Attends scheduled gaming or social events

>   - Paricipates in virtual conversations

Reviewer 2: ```suggestion
  - Participates in conversation
```

Specifying `virtual` is unnecessary and excludes the eventual possibility of irl conversations. It also implies that voice chat wouldn't necessarily count (at least in my reading of this).

Author: I was mentally including irl conversations in "attending social events" . 

Me learning to spell is a whole other issue...

Reviewer 2: Oh, right. I'm good at thinking!

* Active members are noted by the **Zargonaut** role in the group discord.

### Contributing Members

> * A contributing member is one who regularly does one or more of the following

Reviewer 2: `regularly` is pretty vague, which leaves it open to cliquish behavior to decide who can attain that status...

Or maybe this is just a time where being a trusted group means we can be a little more vague?

Author: As discussed, we're keeping `regularly` vague and counting on a "you know it when you see it" approach.

  - Schedules game sessions
  - Schedules social events
  - Brings new games for the group to try
  - Teaches games at gaming events
  - Creates content to aid the group (e.g. rules handouts, character
    illustrations, game storage solutions)

> * Contributing members are noted by the **Adept** role in the group discord.

Reviewer 2: **The Council** was also a proposed role for this as well, did we want to consider that?

We should include a process for ascending to the rank of **Adept** / **The Council** as well. Maybe a petition from a **Zargonaut** and a majority vote by other contributing members?

### New Members
* Any active member in good standing may invite non-member to join social
  or gaming events, but take responsibility for their behavior.
  - In the case of a virtual event, they will be assigned the **Attendee** role
    in the group discord.
  - The **Attendee** role is temporary and will be removed once the event

>     concludes

Reviewer 2: This makes some sense for Zargcon-style events where we'll have multiple voice channels that we want to otherwise keep hidden but it seems like overkill for "I want my buddy to join lunch games today".

I could see us having to mess around with roles and permissions just to get into a voice chat being a disincentive to join up.

Reviewer: My point was basically that if it's NOT temporary then it's little different than full zargonaut

Author: To more explicitly state what I was saying: `Attendees` would note have access to all the text channels, we'd make a single one they could chat in and just give them access to the voice channels for simplicity.

Good humored, cooperative, competitive, consensual torture, for the purposes of
exploring strategies, design, progress and history of gaming

## Membership
There is no single leader of the Zargon community. Zargon is a community run
effort. All members are encouraged to participate in this effort.

The following guidelines are intended to allow us to maintain an inclusive
environment where all members feel comfortable gaming with any other member of

> the group rather than fracturing in cliques. They are enumerated in detail so

Reviewer 3: ```suggestion
the group rather than fracturing into cliques. They are enumerated in detail so
```

as to mitigate subjectivity in role assignment.

### Obligations
* Members are expected to maintain basic personal hygiene.
* Members are expected to be gracious and respect additional restrictions of
  venue
    - e.g. no smoking inside; no young children; allergy considerations.

> ### Active Members

Reviewer 3: We need to add a section on inactive members as well.  Like if you're inactive for 1 yr+ you lose **Adept** status.  Or whatever other details need to be ironed out.

> * An active member is one who regularly does one or more of the following

Reviewer 4: ```suggestion
* An active member is one who regularly does one or more of the following:
```

  - Attends and particpates in scheduled gaming
  - Attends and participates in social events
  - Participates in virtual group conversations (e.g. text, voice, video)
* Active members are noted by the **Zargonaut** role in the group discord.

### Contributing Members

> * A contributing member is one who regularly does one or more of the following

Reviewer 4: ```suggestion
* A contributing member is one who regularly does one or more of the following:
```

  - Schedules game sessions
  - Schedules social events
  - Brings new games for the group to try
  - Teaches games at gaming events

>   - Creates content to aid the group (e.g. rules handouts, character

Reviewer 3: I would add does work on Zargon infrastructure.  E.g. contributes to the website, server administration, commits to the bot on discord, etc.

Reviewer 2: :+1:

    illustrations, game storage solutions)
* An active member who feels they are making sufficient contributions should
  bring this to the attention of one of the contributing members.
* Contributing members are noted by the **Adept** role in the group discord.

### New Members

> * Any active member in good standing may invite non-member to join social
>   or gaming events, but take responsibility for their behavior.

Reviewer 4: ```suggestion
* Any active member in good standing may invite a non-member to join social
  or gaming events, but please take responsibility for their behavior.
```

Reviewer: Personally I would change `please` to `must` there.

Reviewer 3: +1 to must.  Maybe "should" if we want it to sound more polite.  But the word "please" moves the sentence from the third person to the second person imperative.

Reviewer 4: i chose "should" because it's more polite sounding.

Author: Keeping both off to keep the tone better.

  - In the case of a virtual event, they will be assigned the **Attendee** role
    in the group discord.
  - The **Attendee** role is temporary and will be removed once the event

>     concludes

Reviewer 4: ```suggestion
    concludes.
```

heh

* Any active member in good standing may nominate a previously-introduced
  individual for full membership.

> * A nominated individual will then be considered as follows

Reviewer 4: ```suggestion
* A nominated individual will then be considered as follows:
```

  venue
    - e.g. no smoking inside; no young children; allergy considerations.

### Active Members
* An active member is one who regularly does one or more of the following:
  - Attends and particpates in scheduled gaming
  - Attends and participates in social events
  - Participates in virtual group conversations (e.g. text, voice, video)
* Active members are noted by the **Zargonaut** role in the group discord.

### Contributing Members
* A contributing member is an active member who also regularly does one or more
 of the following:
  - Schedules game sessions
  - Schedules social events
  - Brings new games for the group to try
  - Teaches games at gaming events
  - Creates content to aid the group (e.g. rules handouts, character
    illustrations, game storage solutions)
  - Contributes to group infrastructure (commits to websites or bots, server
    administration, etc.)
* An active member who feels they are making sufficient contributions should
  bring this to the attention of one of the contributing members.
* Contributing members are noted by the **Adept** role in the group discord.

### New Members

> * Any active member in good standing may invite non-member to join social

Reviewer 4: ```suggestion
* Any active member in good standing may invite a non-member to join social
```

i think we might've list an `a` there?

  advances
* Trolling, insulting/derogatory comments, and personal or political attacks
* Public or private harassment

> * Publishing another member's private information, such as a physical or
>   electronic address, without explicit permission

Reviewer 2: ```suggestion
* Publishing someone's private information, such as a physical or electronic
  address, without explicit permission
```

no doxxing anyone, not just members
